### PR TITLE
Merge pull request #34 from Aggregate-Genius/header-and-announcements…

### PR DESCRIPTION
**Pull request by agenius-mohammed-ali**
…-bar

- Updated NEWS file

### CODE REVIEW

**Review by cb4ds — APPROVED** [review]

File: inst/fw_templ/p_example/ui_body.R [hunk 1]
```diff
@@ -428,6 +428,11 @@ style_guide_box <- box(
     tags$li("'periscope_style.yaml' contains the most important variables with documentation explaining what each variable affects and possible values for it"),
     tags$li("Another method to update application themes or any controls in CSS is by updating 'www/css/custom.css' file"),
     tags$li("The below widget explains some of the variables and how they affect the application"),
+    br(),
+    br(),
+    tags$b(tags$i("** Note: Updating app theme might take few seconds")),
```
Comment (agenius-jennifer) [inline inst/fw_templ/p_example/ui_body.R:433]: I suggest adding "a" before "few seconds" (i.e. "might take a few seconds")

File: inst/fw_templ/p_example/ui_body_no_left_sidebar.R [hunk 2]
```diff
@@ -411,6 +411,11 @@ style_guide_box <- box(
     tags$li("'periscope_style.yaml' contains the most important variables with documentation explaining what each variable affects and possible values for it"),
     tags$li("Another method to update application themes or any controls in CSS is by updating 'www/css/custom.css' file"),
     tags$li("The below widget explains some of the variables and how they affect the application"),
+    br(),
+    br(),
+    tags$b(tags$i("** Note: Updating app theme might take few seconds")),
```
Comment (agenius-jennifer) [inline inst/fw_templ/p_example/ui_body_no_left_sidebar.R:416]: I suggest adding "a" before "few seconds" (i.e. "might take a few seconds")

File: inst/fw_templ/p_example/ui_body.R (hunk 1 above)
Comment (agenius-mohammed-ali) [reply inst/fw_templ/p_example/ui_body.R:433]: done

File: inst/fw_templ/p_example/ui_body_no_left_sidebar.R (hunk 2 above)
Comment (agenius-mohammed-ali) [reply inst/fw_templ/p_example/ui_body_no_left_sidebar.R:416]: done

**Pull request merged**